locksmith/config/config.js

```diff
@@ -0,0 +1,26 @@
+require('dotenv').load()
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOSTNAME,
+    dialect: 'postgres',    
+  },
+  test: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOSTNAME,
+    logging: false, 
+    dialect: 'postgres',      
+  },
+  production: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
```

Comment:
in order to prepare for using docker or kubernetes secrets, which are always represented as a file in a ramfs, I'd add some further logic here.

top of the file:

```js
const fs = require('fs')
```

then:

```js
    password: process.env.DB_PASSWORD_FILE ? fs.readFileSync(process.env.DB_PASSWORD_FILE) : process.env.DB_PASSWORD
```

Then we will be golden

Reply:
unless, of course, you plan to inject the .env file into the locksmith root directly, which is possible, but a bit magic-y. Docker secrets live in /run/secrets by default